# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../djv/djv-0.8.2-fix-docdir.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Fix install destination for documentation

Without this patch djv is installing documentation into $datadir/doc.

--- djv-0.8.2/doc/CMakeLists.txt.orig	2011-04-15 16:13:54.960077822 +0200
+++ djv-0.8.2/doc/CMakeLists.txt	2011-04-15 16:14:27.208077806 +0200
@@ -33,7 +33,7 @@
 
 install(
   FILES ${src}
-  DESTINATION doc
+  DESTINATION doc/djv
 )
 
 if (APPLE)
